src/test/java/org/openapitools/jackson/nullable/JsonNullableValueExtractorTest.java

```diff
@@ -76,7 +75,7 @@ public void testUnwrapList() {
         assertEquals(3, validationResult.size());
         assertTrue(validationResult.stream().anyMatch(c -> c.getPropertyPath().toString().equals("wheels[1].screws") && c.getConstraintDescriptor().getAnnotation().annotationType().getSimpleName().equals("NotNull")));
         assertTrue(validationResult.stream().anyMatch(c -> c.getPropertyPath().toString().equals("wheels[3].screws") && c.getConstraintDescriptor().getAnnotation().annotationType().getSimpleName().equals("NotNull")));
-        assertTrue(validationResult.stream().anyMatch(c -> c.getPropertyPath().toString().equals("persons[0].role") && c.getConstraintDescriptor().getAnnotation().annotationType().getSimpleName().equals("NotNull")));
+        assertTrue(validationResult.stream().anyMatch(c -> c.getPropertyPath().toString().equals("persons[].role") && c.getConstraintDescriptor().getAnnotation().annotationType().getSimpleName().equals("NotNull")));
```

Comment:
@MelleD this however was one of our inital reasons for PR #35: To get the validation to properly return the invalid item in the property path. Why is the item number missing now? Can we fix this and have it return the item number as for the wheels?

Reply:
No it's normal behavior of bean validation cause it's a Set

Same like:
`  Set<@Valid Person> persons = new HashSet<>();`

It's not related to JsonNullable